python/servo/package_commands.py

```diff
@@ -158,7 +158,7 @@ def package(self, release=False, dev=False, android=None, debug=False, debugger=
             now = datetime.utcnow()
 
             print("Creating Servo.app")
-            dir_to_dmg = '/'.join(binary_path.split('/')[:-2]) + '/dmg'
+            dir_to_dmg = '/'.join(dir_to_build) + '/dmg'
```

Comment:
Yes, I have a second commit incoming that does all the `path` tidying - just testing it all to make sure it works again as a blob (and because `servo` currently isn't building on my Mac for some reason).